hledger-web/Hledger/Web/Foundation.hs

```diff
@@ -31,7 +31,9 @@ import Data.Time.Calendar (Day)
 import Network.HTTP.Conduit (Manager)
 import Network.HTTP.Types (status403)
 import Network.Wai (requestHeaders)
-import System.FilePath (takeFileName)
+import System.Directory (XdgDirectory (..), createDirectoryIfMissing,
+                         getXdgDirectory)
```

Comment:
Can you add a `>=1.2.3.0` bound to directory in hledger-web/package.yaml.

Reply:
done. I also updated the .cabal file.